### PR TITLE
Updates FOIA link

### PR DESCRIPTION
**Pull request by emileighoutlaw**
FOIA page has final approval and can be linked on the footer! 🎉 

cc @AmyKort for visibility

### CODE REVIEW

File: fec/fec/templates/base.html [hunk 1]
```diff
@@ -169,7 +169,7 @@
                 <a href="https://www.data.gov/open-gov/">Open government</a>
               </li>
               <li>
-                <a href="https://www.foia.gov/">FOIA</a>
+                <a href="{{ cms_url }}/freedom-information-act/">FOIA</a>
```
Comment (noahmanger) [inline fec/fec/templates/base.html]: This should just be `/freedom-information-act`, no `{{ cms_url}}` needed. (But this is the format to use when adding to the web app).

Comment (emileighoutlaw) [inline fec/fec/templates/base.html]: Oh, cool!

**Comment by noahmanger** [comment]
Can you also add this in the web app?

**Comment by emileighoutlaw** [comment]
Yep! Here's the related PR, Noah: https://github.com/18F/openFEC-web-app/pull/2014

**Comment by noahmanger** [comment]
Replaced by https://github.com/18F/fec-cms/pull/1049

**Pull request closed without merging**